### PR TITLE
more debug rpcs

### PR DESCRIPTION
**Pull request by sinkingsugar**
prefixed debug rpcs with `debug_`
Add more gossipsub related rpcs

### CODE REVIEW

File: beacon_chain/rpc/nimbus_api.nim [hunk 1]
```diff
@@ -119,25 +118,48 @@ proc installNimbusApiHandlers*(rpcServer: RpcServer, node: BeaconNode) =
       raise (ref CatchableError)(
         msg: "Compile with '-d:chronosFutureTracking' to enable this request")
 
-  rpcServer.rpc("getGossipSubPeers") do () -> JsonNode:
+  rpcServer.rpc("debug_getGossipSubPeers") do () -> JsonNode:
```
Comment (mratsim) [inline beacon_chain/rpc/nimbus_api.nim:121]: I think this one needs a deprecation period.

Comment (arnetheduck) [reply beacon_chain/rpc/nimbus_api.nim:121]: hm, we don't have it documented in api.md so should be fine

**Comment by sinkingsugar** [comment]
consensus on this? can we merge it?

File: beacon_chain/rpc/nimbus_api.nim [hunk 2]
```diff
@@ -119,25 +118,48 @@ proc installNimbusApiHandlers*(rpcServer: RpcServer, node: BeaconNode) =
...
       %(
         peerId: $v.peerId,
         score: v.score,
         iWantBudget: v.iWantBudget,
         iHaveBudget: v.iHaveBudget,
         outbound: v.outbound,
         appScore: v.appScore,
-        behaviourPenalty: v.behaviourPenalty
+        behaviourPenalty: v.behaviourPenalty,
+        sendConnAvail: if v.sendConn != nil:
```
Comment (tersec) [inline beacon_chain/rpc/nimbus_api.nim]: just `v.sendConn != nil`, `v.sendConn != nil and v.sendConn.closed` as boolean-returning expressions, etc?

I don't understand what `if true_or_false: true else: false` accomplishes.

Comment (sinkingsugar) [reply beacon_chain/rpc/nimbus_api.nim]: true, will clean it up. I had more logic probably.

File: beacon_chain/rpc/nimbus_api.nim (hunk 1 above)
Comment (arnetheduck) [reply beacon_chain/rpc/nimbus_api.nim:121]: ok, add a compatibility note to changelog

**Review by arnetheduck — APPROVED** [review]

**Pull request merged**